packages/subnav/partials/TitleLink/index.js

```diff
@@ -31,7 +33,7 @@ function TitleLink(props) {
   return (
     <LinkWrap
       Link={Link}
-      className={`title-link brand-${product} ${!Logo ? 'is-text' : ''}`}
+      className={classNames(s.root, s[`brand-${product}`])}
```

Comment:
`brand-product` class has been retained here for a special case related to sizing the HCP logo.